ui/connections.js

```diff
@@ -133,10 +139,15 @@ module.exports = function () {
           alert(this.$root.$t('connections.unsupportedConnectionTypeX', { connType: s[0] }))
           return
         }
-        SSB.net.connectAndRemember(this.address, {
-          key: '@' + s[s.length-1] + '.ed25519',
-          type: this.type
-        })
+        if (s[0] == 'dht') {
+          SSB.net.dhtInvite.start(() => {});
```

Comment:
I guess if we want to do the start, we should include the next line in the cb and also handle err. But isn't this already handled in core?